azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/presentation/fragment/calling/banner/BannerViewModel.kt

```diff
@@ -104,7 +104,6 @@ internal class BannerViewModel(private val localizationProvider: LocalizationPro
         } else if ((recordingState == ComplianceState.STOPPED) &&
             (transcriptionState == ComplianceState.STOPPED)
         ) {
-            resetStoppedStates()
```

Comment:
Just removing this might produce new bugs for the case this reset was actually implemented. 
We could test more if you like.